### ISSUE TITLE
Add new devtools icons

### ISSUE THREAD

**Issue by nt1m** [FirefoxUX/photon-icons#27]
<img width="258" alt="screen shot 2018-07-09 at 18 13 02" src="https://user-images.githubusercontent.com/1795135/42465483-01f0016e-83a4-11e8-810b-841b9a0d60c4.png">
<img width="237" alt="screen shot 2018-07-09 at 18 13 09" src="https://user-images.githubusercontent.com/1795135/42465485-0208a980-83a4-11e8-80e8-ab8ea953ce65.png">

All available at:
https://searchfox.org/mozilla-central/source/devtools/client/themes/images

**Comment by aminalhazwani** [comment]
Hey @nt1m! Thanks for sharing this, we believe that most of the icons look Photon but we have doubts on the Settings one: it goes in a different direction as our current settings icon https://design.firefox.com/icons/viewer/#preferences fill vs. outline. I briefly talked with Victoria - she is the designer on devtools - but she told me that someone else drew those icons. Feel free to share the issue with the person who designed those, thanks!

**Comment by nt1m** [comment]
@aminalhazwani I believe that settings icon is just a 2px stroke version of the older devtools settings icon. It looks like the engineer who worked on bug 1461522 [designed it](https://bugzilla.mozilla.org/show_bug.cgi?id=1461522#c16).

**Comment by birtles** [comment]
These icons were already added in 5f2b7431efc331433f67d8f9ae580b7c793f8c1e.

We could use the photon preferences one I guess. It's up to Victoria.

(Also, I need to update the dock to bottom icon since there was some feedback that users found it harder to scan since it has a different width to the other dock icons.)

**Comment by aminalhazwani** [comment]
@birtles yeah, the other piece of feedback was actually regarding the dock to bottom icon width 😉 We do also have a doorhanger icon (disable popup), not up on the icons website yet 😔

**Comment by birtles** [comment]
I filed a bug for fixing the icon width so I'll get to that sometime in the 63 Nightly cycle.

**Comment by birtles** [comment]
Oh, one more thing, I wonder if we should use the "Open In New" icon for the "Separate window" menu item and drop the "Dock Undock" icon? @violasong Any preference?

**Comment by violasong** [comment]
@birtles Sorry for the delay in responding to this, I was unsure which icon I liked better - but I saw the 'open in new' icon in your bugzilla screenshot and it looked great! One benefit is that it looks more different from the three docking icons than the old separate window icon, which could make it easier to parse as a group, especially in row format.
